app/helpers/activitystreams2_builder_helper.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module Activitystreams2BuilderHelper
+  # Gets a usable name for an account, using display name or username.
+  def account_name(account)
+    account.display_name.empty? ? account.username : account.display_name
```

Comment:
As I've learned, this should be written as `account.display_name.presence || account.username`